### PR TITLE
[US106 Task113] Added Date Range functionality Frontend

### PR DESCRIPTION
**Pull request by AdvaitChirmule**

### CODE REVIEW

**Comment by AdvaitChirmule** [comment]
Screenshot:

image

File: react-ui/src/project/index.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const Project = () => {
     const [isCycleTime, setIsCycleTime] = useState(false);
     const [isDevFocus, setIsDevFocus] = useState(false);
     const [isCruft, setIsCruft] = useState(false);
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
```
Comment (shahih25) [inline react-ui/src/project/index.js:27]: You need to pass the start date and end date in the request. Check where **axios** call is being made and see how project slug is being passed.

File: react-ui/src/project/index.js [hunk 2]
```diff
@@ -78,28 +80,30 @@ const Project = () => {
             method: "post",
             url: `http://localhost:8000/${metric}`,
             data: {
-                projectslug: project
+                projectslug: project,
```
Comment (shahih25) [inline react-ui/src/project/index.js:83]: Can you please confirm all the other metrics work fine after adding this change?

Comment (shahih25) [reply react-ui/src/project/index.js:83]: And add an evidence with the dates being passed.

Comment (AdvaitChirmule) [reply react-ui/src/project/index.js:83]: Yes, here are the screenshots:

With the dates added:
image

Without the dates added:
image

**Pull request merged**